app/screens/index.js

```diff
@@ -39,6 +40,7 @@ import SelectServer from 'app/screens/select_server';
 import SelectTeam from 'app/screens/select_team';
 import Settings from 'app/screens/settings/general';
 import Thread from 'app/screens/thread';
+import ClockDisplay from 'app/screens/clock_display';
```

Comment:
can we move this up, I know is silly but I would like to keep the imports sorted by name